### DISCUSSION TITLE
Show issue number on new beta projects

### DISCUSSION THREAD

**Discussion by xbc5** [community/community#12879] [Projects and Issues] [answered]
It would be nice to see the issue number at a glance (instead of hovering) which makes it slightly easier to quickly context switch and close issues with commit messages.

Example:
> 10. ( ^ ) My Issue  # 1

**Comment by futurepaul** [comment]
Not an answer, but I actually don't see issue numbers in the List view at all, even on hover (Safari Mac)

**Comment by nkolev92** [comment]
When working on the backlog, the issue # is just as critical as the title.

**Comment by yousan** [comment]
👍 
We need Issue number because project has many issues.

**Comment by andyjones81** [comment]
Yep, I've just accidentally checked in an item against the line item number instead of the issue number. Seems a basic thing it should have. 

As a workaround, I added a new Issue # field and now add the number into it when converting a draft item to an issue. Working for now, but number needs adding to the end of the title or something as a basic feature.

**Comment by kfogel** [comment]
See #8491 about this (same problem, just an earlier ticket number).

**Comment by rileybroughten** [comment, marked as answer]
Making this as a duplicate of [#5904](https://github.com/community/community/discussions/5904), keep an eye on the original discussion post for updates! 👀
